README.markdown

```diff
@@ -314,6 +314,13 @@ or
 
 See [more_set_input_headers](#more_set_input_headers) for more details.
 
+Wildcard `*` can also be used to specify a header name pattern. For example, the following directive effectively clears *any* input headers starting by "`X-Hidden-`":
```

Comment:
I think we only allow `*` as suffix for now, right?
Better more clear it in the doc?